root/frontend/src/App.jsx

```diff
@@ -55,32 +55,137 @@ class App extends Component {
       showAlert: ['success', 'Semester added to flowchart. Drag any classes into the new semester.'], 
       Semesters: [...this.state.Semesters, semester] 
     });
+    /*/ 
+      This is where the "Add Semseter" API call exists 
+    /*/
+
+    // Use the split() function to split the string into an array
+    // based on the '-' delimiter
+    var semesterArray = semester.split('-');
+    // Store the first element of the array (i.e., "Summer") as lowercase into a variable
+    var season = semesterArray[0].toLowerCase();
+    // Store the second element of the array (i.e., "1") into a variable and cast it to an integer
+    var number = parseInt(semesterArray[1]);
+
+    console.log(season)
+    console.log("\n")
+    console.log(number)
+    console.log("\n")
+
+    let apiURL = "http://localhost:4001/semester"
+    let jsonReq = {
+        "term": season,
+        "year": number,
+        "Courses_list": []
+    }
+    fetch(apiURL, {
+      method: 'POST',
+      headers: {'Content-Type':'application/json'},
+      body: JSON.stringify(jsonReq),
+      mode: 'cors'
+    }).then( (response) => {
+      if (!response.ok) {
+        throw Error(response.statusText);
+      }
+      console.log("hot dog, ", response)
```

Comment:
We may want to make this console.log have a meaningful text, like

```suggestion
      console.log("Received response from the server for CreateSemester, ", response)
```